### DISCUSSION TITLE
Rich Protocol

### DISCUSSION THREAD

**Discussion by Kludex** [Instagram/LibCST#910] [Ideas]
Hey there :wave: 

Would you be open to introduce the Rich Protocol on LibCST? It will help me greatly to debug. :eyes: 

I can open a PR, if you'll accept.

**Comment by zsol** [comment]
Do you mean https://rich.readthedocs.io/en/stable/pretty.html#rich-repr-protocol ? If so, I'm happy to accept a PR

**Reply by Kludex** [reply]
yes :+1: I thought I had added the hyperlink, sorry :sweat_smile:
